indexer/validate_and_format_contacts.cpp

```diff
@@ -14,22 +14,50 @@ static auto const s_badVkRegex = regex(R"(^\d\d\d.+$)");
 static auto const s_goodVkRegex = regex(R"(^[A-Za-z0-9_.]{5,32}$)");
 static auto const s_lineRegex = regex(R"(^[a-z0-9-_.]{4,20}$)");
 
-char const * const kWebsiteProtocols[] = {"http://", "https://"};
-size_t const kWebsiteProtocolDefaultIndex = 0;
+constexpr string_view kFacebook{"contact:facebook"};
+constexpr string_view kInstagram{"contact:instagram"};
+constexpr string_view kTwitter{"contact:twitter"};
+constexpr string_view kVk{"contact:vk"};
+constexpr string_view kLine{"contact:line"};
+
+constexpr string_view fbProfilePhp{"profile.php"};
```

Comment:
According to our docs/CPP_STYLE.md:
> - Compile-time constants must be named in camelCase, starting with a lower-case `k`, e.g. `kCompileTimeConstant` and marked as `constexpr` when possible.

So all constants should be named like kConstant.

Reply:
Ok, didn't see style guide file.
Renamed constants in the file.